packages/calcite-components/src/utils/time.spec.ts

```diff
@@ -33,19 +34,252 @@ describe("formatTimePart", () => {
   });
 });
 
+describe("getLocalizedMeridiem", () => {
+  it("ar", () => {
```

Comment:
What kind of link?

Reply:
An official resource regarding localized meridiem entries. Maybe from CLDR (e.g., https://github.com/unicode-org/cldr-json/blob/d38478855dd8342749f0494332cc8acc2895d20d/cldr-json/cldr-dates-modern/main/es-MX/ca-generic.json#L227)?

Reply:
@annierm18 : is it ok to put links within commented code to the `cldr-json` github repo for reference, or does Esri Intl prefer a different place, such as an Esri-hosted url containing data like the localized meridiems?  See the link in Franco's comment above.

Reply:
@eriklharper from our perspective, it would be okay to link to CLDR JSON as long as there are no security concerns with including an external link cc: @jcfranco